### PR TITLE
Fix System Settings Required Fields (Enable Jira Integration)

### PR DESCRIPTION
**Pull request by FelixHernandez**
**Description**

This feature allows the user save the settings without jira Url when the Enable Jira Integration is not checked.

**Test results**
The form shows the validation message next to the jira url field.
image

### CODE REVIEW

File: dojo/templates/dojo/findings_list_snippet.html
Comment (blakeaowens) [inline dojo/templates/dojo/findings_list_snippet.html:1]: Please remove this change from this PR

Comment (FelixHernandez) [reply dojo/templates/dojo/findings_list_snippet.html:1]: [This is the commit](https://github.com/DefectDojo/django-DefectDojo/pull/8623/commits/328d6c819dbb3d493d8a18eee7cacc035319cca0) that remove these changes @blakeaowens

**Review by blakeaowens — APPROVED** [review]
Looks good to me :)

**Review by grendel513 — APPROVED** [review]

**Review by cneill — APPROVED** [review]

**Review by mtesauro — APPROVED** [review]
Approved

**Pull request merged**